libs/orchestrator-ui-components/src/lib/components/Subscription/SubscriptionActions.tsx

```diff
@@ -0,0 +1,121 @@
+import React, { FC, useState } from 'react';
+import {
+    EuiButton,
+    EuiContextMenuItem,
+    EuiContextMenuPanel,
+    EuiPanel,
+    EuiAvatar,
+    EuiTitle,
+    EuiPopover,
+} from '@elastic/eui';
+import { useSubscriptionActions } from '../../hooks/useSubscriptionActions';
+
+export type SubscriptionActionsProps = {
+    subscriptionId: string;
+};
+
+export const SubscriptionActions: FC<SubscriptionActionsProps> = ({
```

Comment:
`SubscriptionActions` component needs to be covered by both flavours of UI integration test(s) namely Jest snapshot tests and Storybook stories towards avoiding unintentional regressions both in the rendered HTML and the UI itself respectively.

Reply:
Not yet #96